src/components/ShoeOrderForm.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="mt-8">
+    <p class="text-body-2 font-weight-medium">Color Variations :</p>
+    <div class="d-flex mt-2">
+      <ColorVariant color-hex="#000000" />
+      <ColorVariant color-hex="#00FFF0" />
+      <ColorVariant color-hex="#FF0000" />
```

Comment:
@balanejacob Define these in the parent component and pass them as props to this component.